disco/voice/udp.py

```diff
@@ -85,7 +129,105 @@ def send_frame(self, frame, sequence=None, timestamp=None, incr_timestamp=None):
 
     def run(self):
         while True:
-            self.conn.recvfrom(4096)
+            data, addr = self.conn.recvfrom(4096)
+
+            # Data cannot be less than the bare minimum, just ignore
+            if len(data) <= 12:
+                continue
+
+            first, second, sequence, timestamp, ssrc = struct.unpack_from('>BBHII', data)
+
+            rtp = RTPHeader(
+                version=first >> 6,
+                padding=(first >> 5) & 1,
+                extension=(first >> 4) & 1,
+                csrc_count=first & 0x0F,
+                marker=second >> 7,
+                payload_type=second & 0x7F,
+                sequence=sequence,
+                timestamp=timestamp,
+                ssrc=ssrc,
+            )
+
+            # Check if rtp version is 2
+            if rtp.version != 2:
```

Comment:
Same in here, basically anyplace we ignore/throw away the packet in this function we should throw a debug log.